lib/Parser/RegexRuntime.cpp

```diff
@@ -5756,6 +5756,7 @@ namespace UnifiedRegex
         if ((flags & GlobalRegexFlag) != 0) w->Print(_u("global "));
         if ((flags & MultilineRegexFlag) != 0) w->Print(_u("multiline "));
         if ((flags & IgnoreCaseRegexFlag) != 0) w->Print(_u("ignorecase"));
+        if ((flags & DotAllRegexFlag) != 0) w->Print(_u("dotAll"));
```

Comment:
>if ((flags & DotAllRegexFlag) != 0) w->Print(_u("dotAll")); [](start = 8, length = 59)

nit: all of these strings should have a space at the end (like `"global "` above) so that they are spaced out correctly when printed